runner_facebook.sh

```diff
@@ -1,23 +1,25 @@
 #!/usr/bin/env bash
 
-OLD=$PWD
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
 cd $DIR
 
-if [ -d "/usr/local/crawler-venv" ]
-then
-  source /usr/local/crawler-venv/bin/activate
-fi
-
-FOLDER=$(date +"%Y-%m")
-mkdir -p "$OLD/$FOLDER"
+source credentials.txt
+source sites.txt
 
-DAY=$(date +"%d")
-mkdir -p "$OLD/$FOLDER/$DAY/facebook"
+TIME=$(date +"%Y-%m-%d_%H-%M")
+DATE=$(date +"%Y-%m-%d")
 
-TIME=$(date +"%s")
+for site in $sites
+do
+    echo " "
+    echo "************* Current page: $site ***************************************"
+    mkdir -p "../res/$site/"
+   
+    scrapy crawl fb -a email="$FACEBOOK_EMAIL" -a password="$FACEBOOK_PASSWORD" -a page="$site" -a lang="en" -o "../res/$site/posts/$TIME.csv" -a date="$DATE" 
+    
+    scrapy crawl comments -a email="$FACEBOOK_EMAIL" -a password="$FACEBOOK_PASSWORD" -a page="$site" -a lang="en" -o "../res/$site/comments/$TIME.csv" -a date="$DATE" 
 
-source ~/.secrets
```

Comment:
As long as credentials.txt is included in the merge, removing this is recommended.